schemas/fingerprint-server-api.yaml

```diff
@@ -713,6 +718,7 @@ components:
             factoryReset:
               title: WebhookSignalResponseFactoryReset
               type: object
+              additionalProperties: false
               properties:
                 time:
                   title: Time
```

Comment:
Unrelated, but should this description include iOS?

Reply:
What do you mean by iOS? As I see from the description it is an Android specific signal.

Reply:
Oh guys the docs are slightly outdated, iOS also has Factory Reset Detection with the very same structure.